crates/store/src/state.rs

```diff
@@ -691,7 +693,7 @@ impl State {
         let inner_state = self.inner.read().await;
 
         let state_headers = if !include_headers {
-            BTreeMap::<AccountId, AccountStateHeader>::default()
+            BTreeMap::<u64, AccountStateHeader>::new()
```

Comment:
Why was this change necessary?

Reply:
It wasn't, I thought I was rolling it back correctly but I wasn't (`AccountId` here is a newtype for `u64`)